src/commerce-handler.js

```diff
@@ -3,34 +3,38 @@ function CommerceHandler(common) {
 }
 
 CommerceHandler.prototype.logCommerceEvent = function (event) {
-    if (event.EventCategory == 16 && event.ProductAction) {
-        var itemsArray = [];
-        for (var i = 0; i < event.ProductAction.ProductList.length; i++) {
-            var productDict = {}
-            for (var key in event.ProductAction.ProductList[i]) {
-                productDict[key] = event.ProductAction.ProductLgitist[i][key];
+    if (!common.forwardWebRequestsServerSide) {
```

Comment:
Unfortunately our server doesn't return booleans, instead it returns a titlecase string `"False"` or "`True`, and so `!common.forwardWebRequestsServerSide` would always be trying to get the opposite of a truth value, which is `false`.  
What you can do is set this on `initialization.js` to === 'True' (I added a suggestion there too), and then the `common.forwardWebRequestsServerSide` will be the boolean you are expecting.

2nd suggestion.  Rather than making it so this PR looks a lot bigger than it is, do something like:

```
if (common.forwardWebRequestsServerSide) return false;
// previous code unindented which will no longer be considered edited code and inflate the PR line edits
```

Then `return true` at the bottom as normal.